subprojects/docs/src/docs/userguide/workingWithFiles.adoc

```diff
@@ -280,6 +754,8 @@ You can further customize the archive names:
 </sample>
 ++++
 
+You can find all the possible archive task properties in the API documentation for api:org.gradle.api.tasks.bundling.AbstractArchiveTask[], but we have also summarized the main ones in <<archiveTasksNamingProperties>> below.
+
 [[archiveTasksNamingProperties]]
 .Archive tasks - naming properties
 [cols="a,a,a,a", options="header"]
```

Comment:
We should try to convert most tables we come across to a definition list format. For example: https://docs.gradle.org/current/userguide/java_plugin.html#sec:java_tasks as done in https://github.com/gradle/gradle/commit/4ae351f6cf5f725257d13c4a82741ef2ae0e2bf0#diff-cada99fea6a20ffb6871debbd42b6d7f which makes the content respond much better to various widths.

Reply:
I'll look into it, but it _is_ a 4-column table

Reply:
Done